src/main/java/com/blackducksoftware/integration/jira/config/controller/IssueCreationConfigController.java

```diff
@@ -154,6 +154,28 @@ public Response getCommentOnIssueUpdatesChoice(@Context final HttpServletRequest
         return Response.ok(config).build();
     }
 
+    @Path("/project/reviewerchoice")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getProjectRevieweerNotificationsChoice(@Context final HttpServletRequest request) {
+        logger.debug("GET /project/reviewerchoice");
+        final Object config;
+        try {
+            final boolean validAuthentication = isAuthorized(request);
```

Comment:
We can probably now call this method directly in the if statement since it's so small.